packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -134,6 +141,12 @@ const ListFieldText = styled('div')`
   color: ${({ theme }) => theme.custom.gray[800]};
 `
 
+const ListFieldValue = styled(Typography)`
```

Comment:
For consistancy, let's not use Typography, as we generally don't use it in the app.
```suggestion
const ListFieldValue = styled('div')`
```

And then you can remove the `as="p"` and the Typography import.